workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -42,19 +49,72 @@ public static void BuildConfig()
             }
         }
 
-        public static void Launch()
+        public static void LaunchClient()
         {
-            BuildConfig();
+            var command = GetCommand();
+            var commandArgs = GenerateCommandArgs("local worker launch UnityClient default");
 
-            var command = Common.SpatialBinary;
-            var commandArgs = "local launch";
-            
-            if (Application.platform == RuntimePlatform.OSXEditor)
+            var processInfo = new ProcessStartInfo(command, commandArgs)
             {
-                command = "osascript";
-                commandArgs = $"-e 'tell application \"Terminal\"\nactivate\ndo script \"cd {SpatialProjectRootDir} && {Common.SpatialBinary} local launch\"\nend tell'";
+                CreateNoWindow = false,
+                UseShellExecute = true,
+                WorkingDirectory = SpatialProjectRootDir
+            };
+
+            var process = Process.Start(processInfo);
+
+            if (process == null)
+            {
+                Debug.LogError("Failed to start an additional client locally.");
+                return;
             }
 
+            process.EnableRaisingEvents = true;
+            process.Exited += (sender, args) =>
+            {
+                // N.B. This callback is run on a different thread.
+                if (process.ExitCode == 0)
+                {
+                    return;
+                }
+
+                var logPath = Path.Combine(SpatialProjectRootDir, "logs");
+                var latestLogFile = Directory.GetFiles(logPath, "*unityclient.log")
+                    .Select(f => new FileInfo(f))
+                    .OrderBy(f => f.LastWriteTimeUtc).LastOrDefault();
+
+                if (latestLogFile == null)
+                {
+                    Debug.LogError($"Could not find a client log file in {logPath}.");
+                    return;
+                }
+
+                var message = $"Unity Client local launch logfile: {latestLogFile.FullName}";
+
+                if (WasProcessKilled(process))
+                {
+                    Debug.Log(message);
+                }
+                else
+                {
+                    var content = File.ReadAllText(latestLogFile.FullName);
+                    message = $"{message}\n{content}";
```

Comment:
Don't rebind the variable - just create a new one 😄 

Makes the code easier to follow for someone else. You can also inline this in the `Debug` call.

`Debug.LogError($"{message}\n{content}");`

Reply:
Was a leftover from the previous code - perhaps some golden variable for debugging.